src/test/java/Homework14.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+public class Homework14
+{
+
+
+
+
+    @Test
+
+        public static void navigation() {
+
+        WebDriver register = new ChromeDriver();
+        register.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://testpro.io/";
+        register.get(url);
+
+        String registrationLink = "#hel";
+        register.findElement(By.cssSelector(registrationLink)).click();
+
+        String registrationUrl = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(register.getCurrentUrl(), registrationUrl);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/209647390-956e602c-fcba-4f87-8dea-6878e9a4be6f.png)

For example

`public class Homework14 extends BaseTest `

Reply:
I never thought those empty lines were killing me so much. ok. I'll adjust it and see what happens.